### ISSUE TITLE
dropbox out of date

### ISSUE THREAD

**Issue by titpetric** [janeczku/docker-dropbox#5]
I'm running the dropbox docker image and i'm getting the following output:

```
root@dropbox:/# dropbox status
You're using an old version of Dropbox. Please update within the next 12 days to continue using Dropbox.

https://www.dropbox.com/downloading?from_client=True
Up to date
```

I've tried also to `apt-get update ; apt-get install dropbox`, and no changes have been made.

```
root@dropbox:/# apt-get install dropbox
Reading package lists... Done
Building dependency tree
Reading state information... Done
dropbox is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 40 not upgraded.
```

```
root@dropbox:/# apt-cache showpkg dropbox
Package: dropbox
Versions:
2015.10.28 (/var/lib/apt/lists/linux.dropbox.com_debian_dists_jessie_main_binary-amd64_Packages.gz) (/var/lib/apt/lists/linux.dropbox.com_debian_dists_jessie_main_binary-i386_Packages.gz) (/var/lib/dpkg/status)
 Description Language:
                 File: /var/lib/apt/lists/linux.dropbox.com_debian_dists_jessie_main_binary-amd64_Packages.gz
                  MD5: f939f05a439263eadbe01f62e4eb496e


Reverse Depends:
  nautilus-dropbox,dropbox
Dependencies:
2015.10.28 - procps (0 (null)) python-gtk2 (2 2.12) python (2 2.5) libatk1.0-0 (2 1.20.0) libc6 (2 2.4) libcairo2 (2 1.6.0) libglib2.0-0 (2 2.16.0) libgtk2.0-0 (2 2.12.0) libpango1.0-0 (2 1.20.1) lsb-release (0 (null)) nautilus (2 2.16.0) python-gpgme (2 0.1) nautilus-dropbox (0 (null)) nautilus-dropbox (0 (null))
Provides:
2015.10.28 - nautilus-dropbox
Reverse Provides:
```

any ideas?

**Comment by janeczku** [comment]
Hi @titpetric thanks for the info. I upgraded the image to the latest Dropbox v3.14.7.
> docker pull janeczku/dropbox:3.14.7
> docker run -ti --rm janeczku/dropbox:3.14.7

```Console
DBOX_UID (user id) variable not specified, defaulting to 1000..
DBOX_GID (group id) variable not specified, defaulting to 1000..
dropboxd(3.14.7) started...
This computer isn't linked to any Dropbox account...
Please visit https://www.dropbox.com/cli_link_nonce?nonce=XXXXXXX to link this device.
```

**Comment by kopax** [comment]
dropbox current linux version available is 3.16.1.

it's not very efficiant to have an image of this, the launcher should take care of upgrading on boot to have always the latest image.

Otherwise you will have to rebuild pretty often.

**Comment by janeczku** [comment]
@kopax Implemented in 5c42478

**Comment by kopax** [comment]
Wow how fast. Thanks for your help.
